libs/MVS/SceneTexture.cpp

```diff
@@ -151,8 +151,11 @@ struct MeshTexture {
 		typedef TRasterMesh<RasterMesh> Base;
 		FaceMap& faceMap;
 		FIndex idxFace;
+		Image8U mask;
+		bool validFace;
+
 		RasterMesh(const Mesh::VertexArr& _vertices, const Camera& _camera, DepthMap& _depthMap, FaceMap& _faceMap)
-			: Base(_vertices, _camera, _depthMap), faceMap(_faceMap) {}
+			: Base(_vertices, _camera, _depthMap), faceMap(_faceMap), validFace(true) {}
```

Comment:
no need to intialize validFace here, it will need to be initialized for every raster face